rh-porta/Containerfile

```diff
@@ -27,7 +27,7 @@ RUN dnf -y module enable ruby:${RUBY_VERSION} nodejs:16 mysql:8.0 \
 # install bundler
 RUN set -x \
     && BUNDLER_VERSION=$(awk '/BUNDLED WITH/ { getline; print $1 }' Gemfile.lock) \
-    && gem install --no-document bundler:$BUNDLER_VERSION --source $BUNDLER_REPO \
+    && gem install --no-document bundler:$BUNDLER_VERSION \
```

Comment:
how does it know where to get it from?

Reply:
Well, this isn't a hermetic build **yet**, so I guess it's from rubygems.org . 

I'll ping you on the PR I open to move to a hermetic build for porta (but that will probably be later - once non-hermetic builds for all other components are onboarded && passing).